typespec-extension/src/main/java/com/azure/cadl/Main.java

```diff
@@ -50,11 +50,11 @@ public class Main {
                 .disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
     }
```

Comment:
Do we want to change the namespace to `package com.azure.typespec;`?

Reply:
java namespace currently is not exposed anywhere, so no change for now (include the name of `CadlPlugin`)